website/scripts/ci/setup_lint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -ex
+
+npm install --legacy-peer-deps
```

Comment:
why `--legacy-peer-deps` ?

Reply:
it seems with react 17+ and npm 7+ the build will fail due to peer dependencies.  This is due to a lot of third party apps not upgraded.
https://weekendprojects.dev/posts/how-to-use-npm-legacy-peer-deps-command/#the---legacy-peer-deps-flag-and-react